### PR TITLE
fix: reformat the style files 

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- Reformatted style files according to various extensions, also pydata-sphinx-theme
- Removed duplicated classes.
- Organized styles under headings.
- A minor change in the appearance of the Sphinx gallery
### before
image
### after
Capture

### CODE REVIEW

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  font-family: "Source Sans Pro";
+  src: url(../fonts/SourceSansPro-Regular.ttf);
+}
+
+:root {
+  /* Ansys specific changes to the theme */
+
+  /*****************************************************************************
+    * Ansys Font family
+    **/
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css:19]: What about using Javadoc style for these comments? This is a common standard understood by tools like Doxygen:

```suggestion
  /**
    * Ansys Font family
    */
```

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    * Colors are defined in rgb string way, "red, green, blue"
+    **/
+  --ansysGold: rgb(255, 183, 27); /* #FFB71B */
+  --ansysBronze: rgb(200, 146, 17); /* #C89211 */
+  --pythonBlue: rgb(57, 114, 161); /* #3972a1 */
+
+  --pst-color-active-navigation: var(--ansysBronze); /* --ansysBronze */
+  --pst-color-navbar-link: rgb(255, 255, 255);
+  --pst-color-navbar-link-hover: var(--ansysBronze); /* --ansysBronze */
+  --pst-color-navbar-link-active: var(--ansysGold); /* --ansysBronze */
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css:46]:
```suggestion
  --pst-color-active-navigation: var(--ansysBronze);
  --pst-color-navbar-link: rgb(255, 255, 255);
  --pst-color-navbar-link-hover: var(--ansysBronze);
  --pst-color-navbar-link-active: var(--ansysGold);
```

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-reformat.css
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-reformat.css:1]: What about `pydta-sphinx-theme-custom.css?`

**Review by jorgepiloto — APPROVED** [review]

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-reformat.css
Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-reformat.css:1]: sure, changing the file names.

**Review by jorgepiloto — APPROVED** [review]

**Pull request merged**